README.md

```diff
@@ -3,23 +3,30 @@
 One-line intro: xarray based verification (accuracy) scoring that can scale with Dask if needed. Pandas supported where possible.
 Why use it: trusted implementations, novel metrics, performance, one-stop-shop.
 
+Included Metrics:
+
+| continuous | probability | statistical tests      |
+| ---------- | ----------- | ----------- |
+| MAE, MSE, RMSE, FIRM | CRPS, FIRM |  Diebold Mariano (with the Harvey 1997 and Hering 2011 modifications) |
```

Comment:
FIRM is primarily categorical rather the probability (but can be applied to continuous and probability of binary event forecasts)

Reply:
Also FIRM is missing in the API documentation https://scores.readthedocs.io/en/doc_experiment/api.html#

Reply:
I've added a categorical column to the table (I expect there will be more categorical scores in due course) in both location.
I've added FIRM to the API documentation.

Reply:
I have udpated the reference to read
Diebold Mariano (with the Harvey et al. 1997 and the Hering and Genton 2011 modifications)